compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2285,7 +2285,7 @@ declare_lint! {
     /// to the same location. Additionally functions may get duplicated if they are instantiated
     /// in different crates and not deduplicated again via LTO.
     pub POINTER_STRUCTURAL_MATCH,
-    Allow,
+    Warn,
     "pointers are not structural-match",
     @future_incompatible = FutureIncompatibleInfo {
         reference: "issue #62411 <https://github.com/rust-lang/rust/issues/70861>",
```

Comment:
There are two related issues here, but in different formats. Are they supposed to refer to the same one?

Reply:
```suggestion
        reference: "issue #70861 <https://github.com/rust-lang/rust/issues/70861>",
```